### DISCUSSION TITLE
Guaranteed Daily Bunch?

### DISCUSSION THREAD

**Discussion by SteveMattan** [ttscoff/bunch-docs#190] [Q&A] [answered]
For historical reasons, I use the application NeoFinder to catalog my images (photography is a hobby). And the version I have does not automatically update the catalogs. So I have Bunch which runs every day at 9 AM to update the catalogs (it does so via AppleScript). This works quite nicely.

However, there are some days that I do not start my day until after 9. And I shut down my Mac every night. So this Bunch does not run. And thus I am sad.

My question: Is there a way to guarantee that  a Bunch will run everyday, other than having it run at startup, or triggering it manually? 

Thanks in advance for any insight. And sorry it if I missed this in the documentation.

SteveM

**Comment by ttscoff** [comment, marked as answer]
No, Bunch can only trigger timers while it's running. For your needs I 
would highly recommend looking into using launchd. An app like 
LaunchControl makes it very easy to set up a recurring task that can 
accommodate system downtime and restarts.

https://soma-zone.com/LaunchControl/

-Brett

**Comment by SteveMattan** [comment]
Thanks. I now have a license to LaunchControl. It is currently 14° F in my neck of the woods, so today is a good day to learn something new.

Thank you again and stay well,

SteveM

**Comment by SteveMattan** [comment]
Well, it took a bit longer than planned, life intervenes sometimes, but I finally had some time to look into launchd and play with LaunchContol. And I'm liking it.

Thank you very much for pointing me to this.

Now back to the rabbit hole ...

:-)
